lib/ansible/modules/cloud/amazon/ecs_task.py

```diff
@@ -66,6 +66,11 @@
         required: false
         version_added: 2.8
         choices: ["EC2", "FARGATE"]
+    task_tags:
```

Comment:
Can this just be tags for consistency with every other AWS module?

Reply:
It should have been.  My syntax highlighter got confused and I did as well.

Thanks for your time reviewing and the suggestions you gave.